src/index.js

```diff
@@ -3,6 +3,8 @@ import ReactDOM from 'react-dom';
 import MarsApp from './components/MarsApp';
 import * as serviceWorker from './serviceWorker';
 import './index.css';
+import photos from './api/photoApiClient';
+
```

Comment:
I assume this is left over from your testing steps? If we're not calling the function anymore we don't need the import statement 😊